src/uct/base/uct_cm.h

```diff
@@ -49,9 +49,28 @@ struct uct_cm {
     uct_base_iface_t iface;
 };
 
+typedef struct uct_cm_sockaddr_wireup_cb {
+    /* Callback to fill the user's private data */
+    uct_sockaddr_priv_pack_callback_t  priv_pack_cb;
+    union {
+        struct {
+            /* On the client side - callback to process an incoming
+             * connection response from the server */
```

Comment:
`response` -> `request`?

Reply:
on the client side - it waits for a response from the server (a response to the client's connection request)